Tasks/Common/azure-arm-rest/azure-arm-app-service-kudu.ts

```diff
@@ -390,7 +390,6 @@ export class Kudu {
         httpRequest.method = 'POST';
         httpRequest.uri = this._client.getRequestUri(`/api/command`);
         httpRequest.headers = {
-            'Content-Type': 'multipart/form-data',
```

Comment:
Do we need to explicitly pass "application/json" as the content-type ?

Reply:
done [here](https://github.com/Microsoft/azure-pipelines-tasks/blob/9bdbe5bb46ee71bb1e180155bd5511699d0840da/Tasks/Common/azure-arm-rest/azure-arm-app-service-kudu.ts#L23).